XamarinCommunityToolkit/Behaviors/AttachedNumericValidationBehavior.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Behaviors
+{
+    public static class AttachedNumericValidationBehavior
```

Comment:
I think we should create a real "Behavior" but preserve such attached property there.

All logic should be kept in real Behavior. This property should just manage if the behavior is added or not (Just my mind)

Reply:
I can remove this file if necessary.  I included this static class because I had used this attached behavior a long time ago but not anytime recently.  Perhaps the other NumericValidationBehavior that inherits from BaseBehavior gets the job done just as good.